src/app.ts

```diff
@@ -402,6 +417,16 @@ export function newApp({
           authMiddlewares.bearerSession
         );
       }
+
+      // eslint-disable-next-line @typescript-eslint/no-use-before-define
+      registerGeoAPIRoutes(
```

Comment:
Do we need to mount this sub-path after a condition on a feature flag?

Reply:
Yep, cause at the moment this feature must be enabled only for CGN.